### PR TITLE
Bugfix/gap 2185 unblock 

### PR DESCRIPTION
**Pull request by john-tco**
This reverts commit 50ab3b6d466d8b4015bc42ece76ecb4a97fade5c.

pr moved here https://github.com/cabinetoffice/gap-find-admin-backend/pull/38 (wrong branch prefix)

### CODE REVIEW

File: src/main/java/gov/cabinetoffice/gap/adminbackend/dtos/ValidateSessionsRolesRequestBodyDTO.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package gov.cabinetoffice.gap.adminbackend.dtos;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Data
```
Comment (ryan-tco) [inline src/main/java/gov/cabinetoffice/gap/adminbackend/dtos/ValidateSessionsRolesRequestBodyDTO.java:9]: Data gives getters and setters

Comment (dominicwest) [reply src/main/java/gov/cabinetoffice/gap/adminbackend/dtos/ValidateSessionsRolesRequestBodyDTO.java:5]: Records > lombok

File: src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/LoginController.java [hunk 2]
```diff
@@ -58,7 +58,7 @@ public ResponseEntity<String> login(HttpServletRequest httpRequest) {
             SecurityContextHolder.getContext()
                     .setAuthentication(new UsernamePasswordAuthenticationToken(
                             new AdminSession(this.grantAdminId, this.funderId, "Test", "User", this.funderName,
-                                    emailAddress),
+                                    emailAddress, null),
```
Comment (dominicwest) [inline src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/LoginController.java:61]: If we've signed in locally wouldn't we want valid roles here 👀

Comment (john-tco) [reply src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/LoginController.java]: Discussed offline - going to remove this block previously used for skipping cola auth locally

**Pull request closed without merging**